Fluid/FiltersCollection.cs

```diff
@@ -6,7 +6,18 @@ namespace Fluid
 {
     public class FilterCollection
     {
-        private Dictionary<string, AsyncFilterDelegate> _delegates = new Dictionary<string, AsyncFilterDelegate>();
+        private readonly Dictionary<string, AsyncFilterDelegate> _delegates;
+
+        public FilterCollection(int capacity = 0)
+        {
+            _delegates = new Dictionary<string, AsyncFilterDelegate>(capacity);
+        }
+
+        public int Count => _delegates.Count;
+
+#if NETSTANDARD2_1
+        public void EnsureCapacity(int capacity) => _delegates.EnsureCapacity(capacity);
```

Comment:
Where is it used?

Reply:
Not yet, but profile showed Resize calls because of adding items dynamically in Orchard, this allows triggering predicted resize before adding